linter/rules.go

```diff
@@ -15,6 +15,7 @@ const (
 	BACKEND_SYNTAX                       = "backend/syntax"
 	BACKEND_DUPLICATED                   = "backend/duplicated"
 	BACKEND_NOTFOUND                     = "backend/notfound"
+	BACKEND_PROBER_CONFIGURATION         = "backend/prober-configuration"
```

Comment:
I think adding a reference for this rule would be a good idea. 
If you check the example that Fastly provides in this [page](https://developer.fastly.com/reference/vcl/declarations/backend/) you can see that they provide an initial value less than the threshold which is confusing tbh for the user

Reply:
What you get is:

```
sotiris.nanopoulos@C02CN3CMML7K falco % ./dist/falco-darwin-amd64 -v examples/default01.vcl
❗️ [WARNING] healthcheck initial value is lower than the threshold. Backend will start as unhealthy (backend/syntax)
in /Users/sotiris.nanopoulos/dev/falco/examples/default01.vcl at line 16, position 4
 15|  .max_tls_version = "1.2";
 16|  .probe = {
       ^^^^^
 17|    .request = "GET /status/200 HTTP/1.1" "Host: httpbin.org" "Connection: close";
See document in detail: https://developer.fastly.com/reference/vcl/declarations/backend/
```

Reply:
I think fastly example is just wrong in this case so it is hard to provide a reference example but I have observed this anecdotally